code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -257,15 +257,21 @@ datum/preferences
 				preview_icon.Blend(markings_s, ICON_OVERLAY)
 
 
-		var/icon/eyes_s = new/icon("icon" = 'icons/mob/human_face.dmi', "icon_state" = current_species ? current_species.eyes : "eyes_s")
-		eyes_s.Blend(rgb(r_eyes, g_eyes, b_eyes), ICON_ADD)
+		var/icon/face_s = new/icon()

```

Comment:
This'll fix your problem with the preview icon being screwy, part 1

Should be `var/icon/face_s = new/icon("icon" = 'icons/mob/human_face.dmi', "icon_state" = "bald_s")`